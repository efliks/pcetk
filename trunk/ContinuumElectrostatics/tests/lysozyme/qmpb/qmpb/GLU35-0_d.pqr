../0/site_GLU_35_d.pqr