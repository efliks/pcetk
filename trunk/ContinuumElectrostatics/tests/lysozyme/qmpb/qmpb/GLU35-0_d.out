Starting /home/mikolaj/local/bin/my_3diel_solver for molecule named GLU35-0_d
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (-0.919373, 20.3963, 19.0418)) 121 2
Finer level:    (3.757, 23.665, 17.308) 101 1
Finer level:    (3.757, 23.665, 17.308) 101 0.5
Finest level:   (3.757, 23.665, 17.308) 101 0.25


Self energy of GLU35-0_d in three dielectric environment = 330.671
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of GLU35-0_d with background charge set in three dielectric environment = -3.0909
(probably in kcal/mole)



Interaction energies of GLU35-0_d with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : -0.028661
0   1 : -0.000459329
1   0 : -0.00170995
1   1 : 0.0360165
2   0 : -0.0171162
2   1 : -0.00034539
3   0 : -0.0670401
3   1 : -0.0122442
3   2 : 0.00789434
3   3 : 0.0565819
4   0 : -0.0114089
4   1 : 0.0412204
5   0 : -0.00901627
5   1 : 0.0678731
6   0 : 0.032534
6   1 : 0.174907
7   0 : -0.160431
7   1 : -0.00719298
8   0 : 407.633
8   1 : 661.342
9   0 : 0.0213564
9   1 : 0.270556
10   0 : 0.00293347
10   1 : 1.21675
11   0 : -0.0097592
11   1 : 0.152398
12   0 : 0.000306248
12   1 : 0.10504
13   0 : -0.000973583
13   1 : 0.0320273
14   0 : -0.0629334
14   1 : -0.00417256
15   0 : -0.0308503
15   1 : -0.000292251
16   0 : -0.00520042
16   1 : 0.154265
17   0 : -0.110586
17   1 : -0.000344028
18   0 : 0.00109933
18   1 : 0.0420509
Total runtime: 7.92 s user 0.07 s system 
